modules/custom/express_permissions/express_permissions.module

```diff
@@ -811,42 +811,77 @@ function express_permissions_form_user_profile_form_alter(&$form, &$form_state)
  */
 function express_permissions_site_owner_cap(&$form, &$form_state) {
   // Limit of how many of the role we should allow.
-  $limit = variable_get("express_permissions_owner_cap", 4);
-  $addon_limit = variable_get('express_permissions_addon_cap', 3);
+  $site_owners_limit = (int) variable_get("express_permissions_owner_cap", 4);
```

Comment:
I thought this was limited to 5 users. I think it is very, very confusing to exclude test users from this limit since...that is not the limit then. It either needs to say `express_permissions_owner_cap_without_test_owner` or set to the actual limit.

Plus, if someone were to set the variable...and it was me...I would have set it to the limit without thinking about test users.

Reply:
https://github.com/CuBoulder/express/issues/2585